build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
   implementation 'org.flywaydb:flyway-core'
   implementation 'net.rakugakibox.spring.boot:orika-spring-boot-starter:1.8.0'
   implementation 'org.postgresql:postgresql'
```

Comment:
commons-io:commons-io or rather org.apache.commons.io.FilenameUtils would no longer be required by applying the commentary listed below